code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KalyanovDmitryTest.java

```diff
@@ -0,0 +1,60 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+
+import static org.junit.Assert.*;
+
+public class KalyanovDmitryTest {
+    private final double delta = 0.001;
+
+    @Test
+    public void createComplexNumEmptyConstructor() {
+        ComplexNumber z1 = new ComplexNumber();
+        ComplexNumber z2 = new ComplexNumber(0, 0);
+        assertEquals(z2, z1);
+    }
+
+    @Test
+    public void getIm() {
+        ComplexNumber z1 = new ComplexNumber(342, 36483);
+        assertEquals(z1.getImaginary(), 36483, delta);
+    }
+
+    @Test
+    public void getRe() {
+        ComplexNumber z1 = new ComplexNumber(342, 36483);
+        assertEquals(z1.getReal(), 342, delta);
```

Comment:
Pls, check assertEquals function: 
`assertEquals(double expected, double actual, double delta)`
Could you swap arguments?

Reply:
Thanks, issue was resolved.